docs/src/tutorial-first-steps.md

```diff
@@ -54,27 +55,37 @@ Notably, we haven't had to write explicit assertions in our proof harness: by de
 ### Getting a trace
 
 By default, Kani only reports failures, not how the failure happened.
-This is because, in its full generality, understanding how a failure happened requires exploring a full (potentially large) execution trace.
-Here, we've just got some nondeterministic inputs up front, but that's something of a special case that has a "simpler" explanation (just the choice of nondeterministic input).
+In this running example, it seems obvious what we're interested in (what `x`?) because we just have one unknown input at the start (similar to the property test), but that's something of a special case.
```

Comment:
```suggestion
In this running example, it seems obvious what we're interested in (the value of `x` that caused the failure) because we just have one unknown input at the start (similar to the property test), but that's kind of a special case.
```